pages/k8s/how-to-cos-lite.md

```diff
@@ -0,0 +1,149 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Integrating COS Lite with Charmed Kubernetes"
+  description: Integrating COS Lite with Charmed Kubernetes
+keywords: Cloud, cluster, observability
+tags: [observability, operating]
+sidebar: k8smain-sidebar
+permalink: how-to-cos-lite.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+**Charmed Kubernetes** includes the standard Kubernetes dashboard for
+monitoring your cluster. However, it is often advisable to have a monitoring
+solution which will run whether the cluster itself is running or not. It may
+also be useful to integrate monitoring into existing setups.
+
+To make monitoring your cluster a delightful experience, Canonical provides
+first-class integration between charmed-kubernetes and COS Lite. This guide
```

Comment:
Let's add this info somewhere please: COS stands for Canonical Observability Stack